jekyll/_cci2/server/v4.1/air-gapped-installation/phase-5-configure-circleci-runner.adoc

```diff
@@ -0,0 +1,138 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Phase 5 - Configuring CircleCI Runner
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to configure CircleCI Runner to run with CircleCI Server in an air-gapped environment.
+:icons: font
+:toc: macro
+:toc-title:
+
```

Comment:
@circlery @lloydChris Same as above, some context about why we are installing runner at this point would be useful